src/jitter.rs

```diff
@@ -574,8 +574,8 @@ impl JitterRng {
     ///
     /// # Example
     ///
-    /// Use `timer_stats` to run the [NIST SP 800-90B Entropy Estimation Suite]
-    /// (https://github.com/usnistgov/SP800-90B_EntropyAssessment).
+    /// Use `timer_stats` to run the [NIST SP 800-90B Entropy Estimation Suite
+    /// ](https://github.com/usnistgov/SP800-90B_EntropyAssessment).
```

Comment:
Looks like it's better to put the line break inside the link part: `[text](\br dest)`.

See [interactive tutorial on links](http://commonmark.org/help/tutorial/07-links.html).

Reply:
Ok, then let''s use that